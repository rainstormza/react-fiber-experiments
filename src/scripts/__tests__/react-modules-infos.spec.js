import reactModulesInfos from "../react-modules-infos";

describe("react-modules-infos", () => {
  const retrievedReactModulesInfos = reactModulesInfos();
  it("result should match snapshot", () => {
    expect(retrievedReactModulesInfos).toMatchSnapshot();
  });
});
