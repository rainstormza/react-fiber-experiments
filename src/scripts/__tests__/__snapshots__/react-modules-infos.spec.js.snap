// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react-modules-infos result should match snapshot 1`] = `
Object {
  "react": Object {
    "description": "React is a JavaScript library for building user interfaces.",
    "version": "16.5.0",
  },
  "react-dom": Object {
    "description": "React package for working with the DOM.",
    "version": "16.5.0",
  },
  "schedule": Object {
    "description": "Cooperative scheduler for the browser environment.",
    "version": "0.3.0",
  },
  "simple-cache-provider": Object {
    "description": "A basic cache for React applications",
    "version": "0.8.0",
  },
}
`;
